### ISSUE TITLE
Search Button click listner

### ISSUE THREAD

**Issue by YogeshTripathi5** [kevalpatel2106/EmoticonGIFKeyboard#18]
where is search button click listener?  how to add  text watcher on edit text view of search button?

**Comment by antonioxtasis** [comment]
yes, where is the listener? @YogeshTripathi5 did you resolve that?

**Comment by kkm** [comment]
+1

**Comment by YogeshTripathi5** [comment]
yep,i got that EmoticonGIFKeyboardFragment is the class search for searchBtn  and to find the textwatcher
EmoticonSearchFragment is the class

**Comment by kevalpatel2106** [comment]
@YogeshTripathi5 I want to understand, why do you need search text change listener? Can you explain the use case for it?
